src/jobs/jobs.module.ts

```
@@ -1,13 +1,16 @@
 import { Module } from '@nestjs/common';
-import { TypeOrmModule } from '@nestjs/typeorm';
+import { TypeOrmModule } from '@nestjs/typeo
+import { JobsService } from './jobs.service';
+import { Job } from './entities/job.entity';
+import { Application } from './entities/application.entity';
 import { Job } from './entities/job.entity';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Job'. Consider to delete it or rename it.

'Job' is defined here:


(lint/suspicious/noRedeclare)